xblock/mixins.py

```diff
@@ -242,6 +249,13 @@ def _field_data(self, field_data):
         warnings.warn("Setting _field_data is deprecated", FieldDataDeprecationWarning, stacklevel=2)
         self._deprecated_per_instance_field_data = field_data
 
+    @property
+    def supports_save(self):
+        """
+        Returns true if this block supports being saved.
+        """
+        return True
```

Comment:
I really like @ormsbee's proposal so that the XBlock class itself provides the desired save. I'll try that out and see if that is sufficient.